Xamarin.Forms.Core/ItemTappedEventArgs.cs

```diff
@@ -4,14 +4,17 @@ namespace Xamarin.Forms
 {
 	public class ItemTappedEventArgs : EventArgs
 	{
-		public ItemTappedEventArgs(object group, object item)
+		public ItemTappedEventArgs(object group, object item, int itemIndex)
```

Comment:
it's unlikely that external code has a reference to this constructor, but in case it does, this is a breaking change. you need to keep the existing ctor (attribute it with `[Obsolete]`) that calls the new one with `-1` as `itemIndex` argument.

Reply:
Good call, will fix it now